tool/tsh/common/play.go

```diff
@@ -128,14 +128,13 @@ func exportSession(cf *CLIConf) error {
 		return trace.Wrap(err)
 	}
 
-	proxyClient, err := tc.ConnectToProxy(cf.Context)
+	clusterClient, err := tc.ConnectToCluster(cf.Context)
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	defer proxyClient.Close()
+	defer clusterClient.Close()
 
-	site := proxyClient.CurrentCluster()
-	evts, errs := site.StreamSessionEvents(cf.Context, *sid, 0)
+	evts, errs := clusterClient.AuthClient.StreamSessionEvents(cf.Context, *sid, 0)
```

Comment:
Maybe:

```suggestion
	eventC, errC := clusterClient.AuthClient.StreamSessionEvents(cf.Context, *sid, 0)
```

Otherwise it looks like an unhandled error.